packages/liveblocks-client/src/utils.test.ts

```diff
@@ -6,8 +6,25 @@ import {
   getTreesDiffOperations,
   findNonSerializableValue,
   isTokenValid,
+  compact,
 } from "./utils";
 
+describe("compact", () => {
+  it("compact w/ empty list", () => {
+    expect(compact([])).toEqual([]);
+  });
+
+  it("icompact removes nulls and undefined values", () => {
```

Comment:
```suggestion
  it("compact removes nulls and undefined values", () => {
```

Tiny typo

Reply:
Good catch! 🦅👁️ Fixed in 44b63ba1400454046ef86cf2df2634ec5f3ac66f 👍